vignettes/miaSim.Rmd

```diff
@@ -36,14 +36,9 @@ knitr::opts_chunk$set(cache = FALSE,
                         warning = FALSE)
 ```
 
-
 # Introduction
```

Comment:
Would be really good to have some basic examples for all relevant major simulation models (and most important options) in the vignette. Have you checked if we have this? If it is too much to add in this PR, kindly open a new github issue to this package and mention that this should be done (as another PR later).